### PR TITLE
✨ Set granny as obstacle and Niffler collects clasp from granny's purse

### LINKED ISSUE #27: Set granny as obstacle
- Niffler can not walk through granny (set granny immovable)

### LINKED ISSUE #28: Niffler collects the clasp from granny's purse
## Acceptance Criteria ##

- When Niffler touches the purse, the clasp disappears.

### PR DESCRIPTION
**Pull request by cboenke**
Closes #27 
Closes #28

### CODE REVIEW

**Review by RonjaMeinecke — APPROVED** [review]
🚀

File: components/game/Game.tsx [hunk 1]
```diff
@@ -18,12 +18,14 @@ function Game() {
...
     let player;
+    let granny;
+    let purse1;
 
     function preload() {
       this.load.image("street", "/canvasLayer1.svg");
       this.load.image("granny", "/granny.png");
       this.load.image("purse1", "/purse1.svg");
-      //   this.load.image("purse2", "/purse2");
+      this.load.image("purse2", "/purse2.svg");
```
Comment (mewiemers) [inline components/game/Game.tsx]: think about your naming purse1 and purse2 ;-)

**Review by mewiemers — APPROVED** [review]
niffler FTW!🐭

File: components/game/Game.tsx [hunk 2]
```diff
@@ -18,12 +18,14 @@ function Game() {
...
+    let granny;
+    let purseClasp;
 
     function preload() {
       this.load.image("street", "/canvasLayer1.svg");
       this.load.image("granny", "/granny.png");
-      this.load.image("purse1", "/purse1.svg");
-      //   this.load.image("purse2", "/purse2");
+      this.load.image("purseClasp", "/purseClasp.svg");
+      this.load.image("purseNoClasp", "/purseNoClasp.svg");
```
Comment (phlgr) [inline components/game/Game.tsx:28]: Great naming!

File: components/game/Game.tsx [hunk 3]
```diff
@@ -79,6 +84,15 @@ function Game() {
         player.setVelocityX(0) && player.setVelocityY(0);
         player.anims.play("standing", true);
       }
+      this.physics.add.collider(granny, player, function () {
+        console.log("hit?");
```
Comment (phlgr) [inline components/game/Game.tsx]: The console logs should be removed before merging

**Review by phlgr — COMMENTED** [review]
GJ 🚀

**Review by phlgr — APPROVED** [review]
GJ 🚀

**Pull request merged**